src/server/getMergeCommits.ts

```diff
@@ -0,0 +1,46 @@
+const git = require("simple-git");
+
+async function getMergeCommits(req, res, config) {
+  try {
+    const { commitAuthor, commitBranch, commitTime, location } = req.body;
+    const path = config.repos.find((repo) =>
```

Comment:
extract this to a function and add in utils file. Then reuse every where.

Reply:
This function has been modified after adding option page. Will make the required changes there!